src/main/java/org/jboss/threads/Version.java

```diff
@@ -48,11 +50,20 @@ private Version() {
         }
         JAR_NAME = jarName;
         VERSION_STRING = versionString;
-        try {
+        boolean logVersion = AccessController.doPrivileged((PrivilegedAction<Boolean>) Version::shouldLogVersion).booleanValue();
+        if (logVersion) try {
             Messages.msg.version(versionString);
         } catch (Throwable ignored) {}
     }
 
+    private static Boolean shouldLogVersion() {
+        try {
+            return Boolean.valueOf(System.getProperty("jboss.log-version", "true"));
+        } catch (Throwable ignored) {
+            return Boolean.FALSE;
```

Comment:
Should it return `Boolean.TRUE` here?

Reply:
Nah, I prefer to err on the side of "less annoying". :)